src/lib/integration_test_local_engine/node_constants.ml

```diff
@@ -0,0 +1,98 @@
+let default_seed_envs =
+  [ ("DAEMON_REST_PORT", "3085")
+  ; ("DAEMON_CLIENT_PORT", "8301")
+  ; ("DAEMON_METRICS_PORT", "10001")
+  ; ("CODA_LIBP2P_PASS", "") ]
+
+let default_block_producer_envs =
+  [ ("DAEMON_REST_PORT", "3085")
+  ; ("DAEMON_CLIENT_PORT", "8301")
+  ; ("DAEMON_METRICS_PORT", "10001")
+  ; ("CODA_PRIVKEY_PASS", "naughty blue worm")
+  ; ("CODA_LIBP2P_PASS", "") ]
+
+let default_snark_coord_envs ~snark_coordinator_key ~snark_worker_fee =
```

Comment:
Minor naming nit: this file is called `Node_constants`, but this file primarily exports functions to construct various aspects of node configuration (non-constant). Perhaps another name like `Node_config` would be better for this module.